pkg/ansible/runner/internal/inputdir/inputdir.go

```diff
@@ -103,14 +137,34 @@ func (i *InputDir) Write() error {
 
 	// If ansible-runner is running in a python virtual environment, propagate
 	// that to ansible.
-	venv := os.Getenv("VIRTUAL_ENV")
-	hosts := "localhost ansible_connection=local"
-	if venv != "" {
-		hosts = fmt.Sprintf("%s ansible_python_interpreter=%s", hosts, filepath.Join(venv, "bin/python"))
-	}
-	err = i.addFile("inventory/hosts", []byte(hosts))
-	if err != nil {
-		return err
+	ansible_inventory := os.Getenv("ANSIBLE_INVENTORY")
+	if ansible_inventory == "" {
```

Comment:
Can we add a code comment here to describe the logic/what we are attempting to do?